diff --git a/node_modules/react-native-splash-screen/react-native-splash-screen.podspec b/node_modules/react-native-splash-screen/react-native-splash-screen.podspec
index 6c03a7a..3d0b4e4 100644
--- a/node_modules/react-native-splash-screen/react-native-splash-screen.podspec
+++ b/node_modules/react-native-splash-screen/react-native-splash-screen.podspec
@@ -9,7 +9,9 @@ Pod::Spec.new do |s|
   s.author       = 'crazycodeboy'
   s.homepage     = package["homepage"]
   s.license      = package["license"]
-  s.platform     = :ios, "7.0"
+  s.platforms    =  { :ios => "7.0", :tvos => "9.0" }
+  s.ios.deployment_target = '8.0'
+  s.tvos.deployment_target = '9.0'
   s.source       = { :git => "https://github.com/crazycodeboy/react-native-splash-screen", :tag => "v#{s.version}" }
   s.source_files  = "ios/*.{h,m}"
   s.dependency "React"
