diff --git a/node_modules/react-native-localize/RNLocalize.podspec b/node_modules/react-native-localize/RNLocalize.podspec
index f4d3c55..e592113 100644
--- a/node_modules/react-native-localize/RNLocalize.podspec
+++ b/node_modules/react-native-localize/RNLocalize.podspec
@@ -11,7 +11,7 @@ Pod::Spec.new do |s|
   s.authors                   = package["author"]
   s.homepage                  = package["homepage"]
 
-  s.platform                  = :ios, "9.0"
+  s.platform                  = { :ios => "9.0", :tvos => "11.0" }
   s.ios.deployment_target     = "9.0"
   s.tvos.deployment_target    = "11.0"
   s.requires_arc              = true
