diff --git a/node_modules/react-native-version-check/react-native-version-check.podspec b/node_modules/react-native-version-check/react-native-version-check.podspec
index 4a69456..2c734e3 100644
--- a/node_modules/react-native-version-check/react-native-version-check.podspec
+++ b/node_modules/react-native-version-check/react-native-version-check.podspec
@@ -10,7 +10,9 @@ Pod::Spec.new do |s|
 
   s.license      = package["license"]
   s.authors      = { "kimxogus" => "kgyoo8232@gmail.com" }
-  s.platform     = :ios, "7.0"
+  s.platforms    =  { :ios => "7.0", :tvos => "9.0" }
+  s.ios.deployment_target = '8.0'
+  s.tvos.deployment_target = '9.0'
 
   s.source       = { :git => package["repository"]["url"] }
 
