diff --git a/node_modules/react-native-keychain/RNKeychainManager/RNKeychainManager.m b/node_modules/react-native-keychain/RNKeychainManager/RNKeychainManager.m
index 98e45fe..70de44b 100644
--- a/node_modules/react-native-keychain/RNKeychainManager/RNKeychainManager.m
+++ b/node_modules/react-native-keychain/RNKeychainManager/RNKeychainManager.m
@@ -12,7 +12,9 @@
 #import <React/RCTBridge.h>
 #import <React/RCTUtils.h>
 
+#if !TARGET_OS_TV
 #import <LocalAuthentication/LAContext.h>
+#endif
 #import <UIKit/UIKit.h>
 
 @implementation RNKeychainManager
@@ -152,6 +154,7 @@ CFStringRef accessibleValue(NSDictionary *options)
 #define kBiometryTypeTouchID @"TouchID"
 #define kBiometryTypeFaceID @"FaceID"
 
+#if TARGET_OS_IOS
 LAPolicy authPolicy(NSDictionary *options)
 {
   if (options && options[kAuthenticationType]) {
@@ -161,6 +164,7 @@ LAPolicy authPolicy(NSDictionary *options)
   }
   return LAPolicyDeviceOwnerAuthentication;
 }
+#endif
 
 SecAccessControlCreateFlags accessControlValue(NSDictionary *options)
 {
@@ -200,14 +204,14 @@ - (void)insertKeychainEntry:(NSDictionary *)attributes
   SecAccessControlCreateFlags accessControl = accessControlValue(options);
 
   NSMutableDictionary *mAttributes = attributes.mutableCopy;
-
   if (accessControl) {
     NSError *aerr = nil;
+#if TARGET_OS_IOS
     BOOL canAuthenticate = [[LAContext new] canEvaluatePolicy:LAPolicyDeviceOwnerAuthentication error:&aerr];
     if (aerr || !canAuthenticate) {
       return rejectWithError(reject, aerr);
     }
-
+#endif
     CFErrorRef error = NULL;
     SecAccessControlRef sacRef = SecAccessControlCreateWithFlags(kCFAllocatorDefault,
                                                                  accessible,
